### PR TITLE
Use latest version of pypd, and add more actions

### PR DESCRIPTION
**Pull request by warrenvw**
Bumping version to `1.0.0` due to change in API.

### CODE REVIEW

File: actions/launch_incident.py [hunk 1]
```diff
@@ -1,18 +1,42 @@
...
+        try:
+            incident = self.pager.Incident.create(
+                data=payload,
+                add_headers={'from': from_email, },
+            )
+        except BadRequest as e:
+            print(e)
 
-        result = self.pager.Event.create(data=payload)
+        print(incident.json)
```
Comment (warrenvw) [inline actions/launch_incident.py]: I should remove this.

File: actions/launch_incident.py [hunk 2]
```diff
@@ -1,18 +1,42 @@
...
+
         if details is not None:
-            payload['details'] = details
+            payload['body'] = {
+                'details': details,
+                'type': 'incident_body',
+            }
+
+        try:
+            incident = self.pager.Incident.create(
```
Comment (warrenvw) [inline actions/launch_incident.py]: Note that we're now using Incident.create() instead of Event.create().

File: actions/resolve_incident.py [hunk 3]
```diff
@@ -2,21 +2,19 @@
 
 
 class ResolveIncident(PagerDutyAction):
-    def run(self, email, ids):
+    def run(self, from_email, resolution, ids):
         """
         Resolve an incident. The PagerDuty v2 API requires that an incident
```
Comment (warrenvw) [inline actions/resolve_incident.py]: Document "resolution"

File: actions/resolve_incident.py [hunk 4]
```diff
@@ -2,21 +2,19 @@
 
 
 class ResolveIncident(PagerDutyAction):
-    def run(self, email, ids):
+    def run(self, from_email, resolution, ids):
```
Comment (warrenvw) [inline actions/resolve_incident.py]: Document resolution

**Comment by LindsayHill** [comment]
Where are we at with this PR? Ready for merge?

**Comment by warrenvw** [comment]
I'll run some tests before we merge. Note the bump in version to 1.0.0 due to the API changes.

**Comment by LindsayHill** [comment]
No longer valid due to pack rewrite

**Pull request closed without merging**